src/wireviz/Harness.py

```diff
@@ -297,8 +297,8 @@ def output(self, filename: (str, Path), view: bool = False, cleanup: bool = True
         graph.save(filename=f'{filename}.gv')
         # bom output
         bom_list = self.bom_list()
-        with open_file_write(f'{filename}.bom.tsv') as file:
-            file.write(tuplelist2tsv(bom_list))
+        # todo: support user choices of BOM format (probably also graphviz outputs, html outputs)
+        bom_helper.generate_bom_outputs(filename,bom_list,bom_helper.WIREVIZ_TSV, bom_helper.EXCEL_CSV)
```

Comment:
See comment in `bom_helper.py`.

Reply:
Understood, leaving comment for future reference, unless requested otherwise.